src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
```

Comment:
Replace the literal '40' with a named constant such as 'DAILY_RENTAL_RATE' to clearly indicate its purpose.

Reply:
not fixed